code/nordvpn_indicator.py

```diff
@@ -216,19 +289,34 @@ def status_check(self, _):
 
         # Get reported NordVPN IP
         output = self.run_command("nordvpn status")
-        if output != None:
-
+        if output is not None:
+            self.status = output.split('  ')[1].strip()
             matches = re.search(r'(([0-9])+\.){3}([0-9]+)', output)
-            if matches != None:
+            if matches is not None:
                 connected_ip = matches.group(0)
 
-        if connected_ip is None and self.connected is True:
-            self.connected = False
-            self.indicator.update(None, self.connected)
+        self.connected = VPN_STATUS.DISCONNECTED if connected_ip is None else VPN_STATUS.CONNECTED
 
-        if connected_ip != None and self.connected is False:
-            self.connected = True
-            self.indicator.update(None, self.connected)
+    def get_status(self):
+        """
+        Returns the current status of the VPN connection as a string
+        """
+        return self.status
+
+    def is_connected(self):
+        """
+        Returns VPN_STATUS instance representing the connection status
+        """
+        return self.connected
+
+    def get_countries(self):
+        """
+        Returns a list of string representing the available countries
+        """
+        countries = self.run_command('nordvpn countries')
+        if countries is None:
+            return []
+        return ''.join(countries).split(' ')[2].split()
```

Comment:
This returns a sorted list of countries
```suggestion
        countries = countries.split(' ')[2].split()
        countries.sort()
        return countries
```